Safewaan/HonkedIn
# Gabbytang/search events

## Trello ticket link

[As a user, I can search for events so that I can find relevant topics.](https://trello.com/c/6tWLDA45)



## Implementation description
* Implemented search feature to the events page
* Users can now type anything in the search bar and get a resulting list events that have either names, description, or creators that match the search term. 
* Created Search and SubmitButton components to be imported into other components. 
* *UPDATED:* Added clear search function which resets the search bar and search results 
   * Moved search button to be aligned with the text bar 
   * Integrated same changes to the Network page
   * Removed SubmitButton file and combined it with the Search file 



## Steps to test
1. Ensure the search field is open to any text value 
2. Clicking the search button should render the list of all events that match the input search term 
3. If nothing is entered in the search bar, the whole list of events in the db should be rendered
4. The search function should match an event by either name, description, or creator name
5. Make sure the "clear search" button works 



## What should reviewers focus on?
* The search function is accurate
* If there are no matches for a search, the page should be empty
